backend/src/xfd_django/xfd_api/tests/test_stats.py

```diff
@@ -539,6 +539,7 @@ def test_vs_trends_success():
         up_host_count=8,
```

Comment:
Window boundary test (API). Seed two ls rows for the same owner: one with "timestamp" at exactly now() - 11 days, one at now() - 11 days - 1 second. Expect the count to include only the in-window row. This proves your DATEADD filter is correct. 

Format drift guard. Seed an ls that is missing "up" or "timestamp" and assert it does not blow up and does not inflate the count. This protects the TRY_CAST path.